math/aliquot_sum.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * \brief Program to return the [Aliquot Sum](https://en.wikipedia.org/wiki/Aliquot_sum) of a number
+ * 
+ * \details
+ * The Aliquot sum s(n) of a non-negative integer n is the sum of all
+ * proper divisors of n, that is, all the divisors of n, other than itself.
+ * For example, the Aliquot sum of 18 = 1 + 2 + 3 + 6 + 9 = 21
+ */
+
+#include <cassert>
+
+/**
+ * Function to return the aliquot sum of a number
+ * @param num The input number
+ */
+unsigned long long int aliquot_sum(unsigned long long int num) {
```

Comment:
also num could be qualified as const

Reply:
Makes sense, I thought it didn't matter, but I'll add it